tests/microgrid/test_component_data.py

```diff
@@ -53,14 +53,17 @@ def test_inverter_data() -> None:
                     phase_1=electrical_pb2.AC.ACPhase(
                         current=metrics_pb2.Metric(value=12.3),
                         voltage=metrics_pb2.Metric(value=229.8),
+                        power_active=metrics_pb2.Metric(value=3109.8),
                     ),
                     phase_2=electrical_pb2.AC.ACPhase(
                         current=metrics_pb2.Metric(value=23.4),
                         voltage=metrics_pb2.Metric(value=230.0),
+                        power_active=metrics_pb2.Metric(value=3528.3),
                     ),
                     phase_3=electrical_pb2.AC.ACPhase(
                         current=metrics_pb2.Metric(value=34.5),
                         voltage=metrics_pb2.Metric(value=230.2),
+                        power_active=metrics_pb2.Metric(value=3680.1),
```

Comment:
maybe you want to add an assert for these values as well?

Reply:
yes, I do want that. Thanks! Fixed